### PR TITLE
Allow multiple email recipients

### PR DESCRIPTION
**Pull request by marctemp**
https://eaflood.atlassian.net/browse/SFI-1920

### CODE REVIEW

File: ffc-pay-alerts/notify/send-email.js [hunk 1]
```diff
@@ -3,13 +3,17 @@ const {
   notifyApiKey,
   notifyEmailTemplateId,
   notifyEmailAddress
-} = require('./config')
+} = require('../config')
+
+const validateEmail = require('./validate-email')
 
 const NotifyClient = require('notifications-node-client').NotifyClient
 const notifyClient = new NotifyClient(notifyApiKey)
```
Comment (marctemp) [inline ffc-pay-alerts/notify/send-email.js:11]: Should this Notify config be outside this file?

File: test/notify/send-email.test.js [hunk 2]
```diff
@@ -0,0 +1,164 @@
+const flatten = require('flat')
+const env = require('../env')
+const mockContext = require('../mock-context')
+const mockMessage = require('../mock-context')
+const mockReference = require('../mock-reference')
+
+let NotifyClient
```
Comment (simonnineteen90) [inline test/notify/send-email.test.js]: does this need to be camelCase?

File: test/notify/send-email.test.js [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+    validateEmail.mockRejectedValue(new Error('must be a string'))
+
+    const wrapper = async () => {
+      await sendEmail(mockContext, mockMessage)
+    }
+
+    expect(wrapper).rejects.toThrow(Error)
+  })
+
+  test('should throw Error when validateEmail rejects', async () => {
```
Comment (simonnineteen90) [inline test/notify/send-email.test.js]: this looks like a duplicate test of the one on line 78

File: test/env.js [hunk 4]
```diff
@@ -2,5 +2,6 @@ module.exports = {
   notifyApiKey: 'NOTIFY TEST, TEAM OR LIVE API KEY STRING',
   notifyEmailTemplateId: 'NOTIFY EMAIL TEMPLATE STRING',
   // https://docs.notifications.service.gov.uk/node.html#smoke-testing
-  notifyEmailAddress: 'simulate-delivered@notifications.service.gov.uk'
+  notifyEmailAddress: 'simulate-delivered@notifications.service.gov.uk',
+  notifyEmailAddresses: 'simulate-delivered@notifications.service.gov.uk,simulate-delivered@notifications.service.gov.uk'
```
Comment (simonnineteen90) [inline test/env.js:6]: does this need to be an array as test is failing in `send-emails.test`. See below comment.

`notifyEmailAddresses: ['simulate-delivered@notifications.service.gov.uk', 'simulate-delivered@notifications.service.gov.uk']`

File: test/notify/send-emails.test.js [hunk 5]
```diff
@@ -0,0 +1,109 @@
...
+    expect(sendEmail).toHaveBeenCalledTimes(1)
+  })
+
+  test('should call sendEmail with context, message, emailAddress and reference when 1 array emailAddress is received', async () => {
+    await sendEmails(mockContext, mockMessage, [emailAddresses[0]], mockReference)
+
+    expect(sendEmail).toHaveBeenCalledWith(mockContext, mockMessage, emailAddresses[0], mockReference)
+  })
+
+  test('should call sendEmail with context, message and default emailAddresses and reference when valid context and message are received', async () => {
```
Comment (simonnineteen90) [inline test/notify/send-emails.test.js:75]: this test is failing. I think `test/env.js` needs updating to include the email addresses in an array. see above comment on `test/env.js`.

File: ffc-pay-alerts/notify/send-email.js (hunk 1 above)
Comment (simonnineteen90) [reply ffc-pay-alerts/notify/send-email.js:11]: As discussed, dont think it needs to be split into seperate file.

**Review by simonnineteen90 — COMMENTED** [review]
Minor suggestions and couple changes to tests.
Also currently failing pipeline build.

File: test/notify/send-email.test.js (hunk 3 above)
Comment (marctemp) [reply test/notify/send-email.test.js]: Well picked up -- removed

File: test/notify/send-email.test.js (hunk 2 above)
Comment (marctemp) [reply test/notify/send-email.test.js]: Yup, camelCased

File: test/env.js (hunk 4 above)
Comment (marctemp) [reply test/env.js:6]: This was originally how I defined this. However, whenever you define an environment var in these file there are parsed to string, i.e. `notifyEmailAddresses: ['simulate-delivered@notifications.service.gov.uk', 'simulate-delivered@notifications.service.gov.uk']` would equal `'['simulate-delivered@notifications.service.gov.uk', 'simulate-delivered@notifications.service.gov.uk']'`

**Review by simonnineteen90 — APPROVED** [review]

**Pull request merged**